source/conf.py

```diff
@@ -409,7 +409,7 @@ def setup(app):
 .. raw:: html
 
   <link rel="stylesheet" href="https://media.readthedocs.org/css/sphinx_rtd_theme.css" type="text/css" />
-  <link rel="stylesheet" href="http://developers.invoca.net/en/""" + version+  """/_static/css/custom.css" type="text/css" />
+  <link rel="stylesheet" href="http://developers.invoca.net/en/""" + version +  """/_static/css/custom.css" type="text/css" />
```

Comment:
Yes there is a ticket: [WEB-1960](https://ringrevenue.atlassian.net/browse/WEB-1960). Product has placed higher priority on [WEB-1958](https://ringrevenue.atlassian.net/browse/WEB-1958) which I am currently grooming / starting work on.